pkg/api/api.go

```diff
@@ -272,6 +272,11 @@ type IssueCodeResponse struct {
 	// request.
 	GeneratedSMS string `json:"generatedSMS,omitempty"`
 
+	// Phone is the phone number in which to send the compiled SMS message. This
```

Comment:
Can we standardize on format, E. 164?

I put in utilities to do this recently.

Reply:
There's a utility `CanonicalPhoneNumber`, but I don't see it actually used anywhere.

Reply:
The phone is pulled from the verification code, which already uses your standardized thing. I've updated the doc to reflect that it's E.164.